nixos/modules/services/security/aide.nix

```diff
@@ -0,0 +1,145 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.aide;
+  configFile = pkgs.writeText "aide.conf" ''
+    # The location of the database
+    database=${cfg.database_url}
+
+    # The location of the log file
+    log_file=/var/log/aide/aide.log
```

Comment:
I can't find the log_file option in the manpage, trying to evaluate the configuration with this option results in an error (unknown config option: 'log_file'), should this maybe be [`report_url`](https://github.com/aide/aide/blob/v0.18.x/doc/aide.conf.5#L171)?